docs/profiling-workflow-corefx.md

```diff
@@ -18,6 +18,12 @@
       - [Analyzing the Results](#Analyzing-the-Results)
       - [Viewing Source Code](#Viewing-Source-Code)
       - [Identifying Regressions](#Identifying-Regressions)
+  - [VTune](#VTune)
```

Comment:
This should have a separate section or at least briefly call out `AMD uProf` which is the similar program for AMD machines.

Reply:
good point, we should definitely have AMD uProf section

@tannergooding do I remember correctly from your presentation that I can run AMD uProf on Intel machine? So I should be able to create similar docs myself without need to access AMD machine?

Reply:
Yes, they should work for the most part (VTune on AMD and uProf on Intel); but various options won't be supported and you may not get as accurate traces/etc.

Reply:
If it isn't working well enough, I can cover the same example for AMD uProf and modify the doc after this PR is merged (just create an issue an assign it to me)

Reply:
@tannergooding excellent! Since I am now working on the PerfCollect and dotnet trace doc I am going to create a separate issue and assign you. Thanks!